docs/quickstart.rst

```diff
@@ -118,6 +119,8 @@ and models.py. You can also do the models a folder and spread your tables there.
     app = Flask(__name__)
     app.config['DEBUG'] = True
     app.config['SECRET_KEY'] = 'super-secret'
+    # Bcrypt is set as default SECURITY_PASSWORD_HASH, which requires a salt
+    app.config['SECURITY_PASSWORD_SALT'] = uuid.uuid4().hex
```

Comment:
I am not sure if we want to generate in examples. I have seen people adding the same code to their repo and then complaining that the site is not working after restart.